web/src/app/util/safeURL.ts

```diff
@@ -2,6 +2,12 @@
 //
 // It tries to determine if the label is misleading.
 export function safeURL(url: string, label: string): boolean {
+  if (url.startsWith('mailto:')) {
+    const email = url.split(':')[1]
```

Comment:
`:` is allowed in the local part of an address if it's in a quoted string, which could break this
```suggestion
    const email = url.substr(7)
```